requirements.txt

```diff
@@ -0,0 +1,34 @@
+asgiref==3.5.2
```

Comment:
Not sure if we should be adding the requirements.txt

Reply:
fixed the white space issues.
Do you know how to ignore that requirements.txt file so we can go ahead and merge this in?

Reply:
Went ahead and just removed it for now.